Weapon Mods/Development/Starcore_Serpent_Arms_Heavy_Metal-Dev/Data/Scripts/CoreParts/Spinal_Rotary_Ammo.cs

```diff
@@ -113,7 +113,7 @@ partial class Parts
                 MaxIntegrity = 0f, // Blocks with integrity higher than this value will be immune to damage from this projectile; 0 = disabled.
                 DamageVoxels = false, // Whether to damage voxels.
                 SelfDamage = false, // Whether to damage the weapon's own grid.
-                HealthHitModifier = 1001, // How much Health to subtract from another projectile on hit; defaults to 1 if zero or less.
+                HealthHitModifier = 1001, // why?? //How much Health to subtract from another projectile on hit; defaults to 1 if zero or less.
```

Comment:
¯\(◉‿◉)/¯ 
You can zero this out if you want.

Reply:
nah it's funny. I really want to see someone nuke a missile with it sometime